wordpress-mobile/MediaPicker-iOS
# Redesign group picker

Fixes #206 

Implements new UI/UX of group picker.

To test:
 - Start the demo app
 - Open the full screen picker 
 - Check if album selector shows up
 - Select album
 - Select items
 - Check if Select option shows top left
 - Deselect Items
 - Check if Select option shows top left, and if selection zero it disappear
 - Press Select and see if assets are selected
 - Start picker again and press cancel.

@iamthomasbishop, @frosty  some notes:

 - Do you want me to implement the background images on the albums? I can do this by repeating the group image on smaller sizes on the back of the front image. I will use the same image to avoid extra downloads or export of media
- I added the possibility to configure the message for selection, by default it will be Select XX, but it can be configured to Insert, or another message that makes sense for the current use of the picker.
